### PR TITLE
Allow admins to define core sidebar tags in config

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Admins may not want to use the same core functional tags that we do on the sidebar.

### CODE REVIEW

File: app/models/article.rb [hunk 1]
```diff
@@ -236,23 +236,6 @@ def self.active_threads(tags = ["discuss"], time_ago = nil, number = 10)
     stories.pluck(:path, :title, :comments_count, :created_at)
   end
 
-  def self.active_eli5(time_ago)
```
Comment (benhalpern) [inline app/models/article.rb]: While I kept the custom logic on the #help tag, I thought that it wasn't needed in the same way with #ama.

**Comment by ludwiczakpawel** [comment]
No no no..

^ @benhalpern for not removing unused CSS!
